arch/arm/include/stm32/stm32g47xxx_irq.h

```diff
@@ -0,0 +1,204 @@
+#define STM32_IRQ_HRTIM1TIMD  (STM32_IRQ_FIRST+71)  /* 71: HRTIM1 timer D interrupt */
+#define STM32_IRQ_HRTIM1TIME  (STM32_IRQ_FIRST+72)  /* 72: HRTIM1 timer E interrupt */
+#define STM32_IRQ_HRTIM1FLT   (STM32_IRQ_FIRST+73)  /* 73: HRTIM1 fault interrupt */
+#define STM32_IRQ_HRTIM1TIMF  (STM32_IRQ_FIRST+74)  /* 74: HRTIM1 timer E interrupt */
+#define STM32_IRQ_CRS         (STM32_IRQ_FIRST+75)  /* 75: CRS (Clock Recovery System) global interrupt */
+#define STM32_IRQ_SAI1        (STM32_IRQ_FIRST+76)  /* 76: SAI4 global interrupt */
+#define STM32_IRQ_TIM20BRK    (STM32_IRQ_FIRST+77)  /* 77: TIM20 Break, Transition error, Index error interrupt */
+#define STM32_IRQ_TIM20UP     (STM32_IRQ_FIRST+78)  /* 78: TIM20 Update interrupt */
+#define STM32_IRQ_TIM20TRGCOM (STM32_IRQ_FIRST+79)  /* 79: TIM20 Trigger, Commutation, Direction Change, and Index interrupt */
+
+#define STM32_IRQ_TIM20CC     (STM32_IRQ_FIRST+80)  /* 80: TIM20 Capture Compare interrupt */
+#define STM32_IRQ_FPU         (STM32_IRQ_FIRST+81)  /* 81: FPU global interrupt */
+#define STM32_IRQ_I2C4EV      (STM32_IRQ_FIRST+82)  /* 82: I2C4 event interrupt */
+#define STM32_IRQ_I2C4ER      (STM32_IRQ_FIRST+83)  /* 83: I2C4 error interrupt */
+#define STM32_IRQ_SPI4        (STM32_IRQ_FIRST+84)  /* 84: SPI4 global interrupt */
+#define STM32_IRQ_AES         (STM32_IRQ_FIRST+85)  /* 85: AES global interrupt */
+#define STM32_IRQ_FDCAN2_0    (STM32_IRQ_FIRST+86)  /* 86: FDCAN2 interrupt 0 */
+#define STM32_IRQ_FDCAN2_1    (STM32_IRQ_FIRST+87)  /* 87: FDCAN2 interrupt 1 */
+#define STM32_IRQ_FDCAN3_0    (STM32_IRQ_FIRST+88)  /* 88: FDCAN3 interrupt 0 */
+#define STM32_IRQ_FDCAN3_1    (STM32_IRQ_FIRST+89)  /* 89: FDCAN3 interrupt 1 */
+
+#define STM32_IRQ_RNG         (STM32_IRQ_FIRST+90)  /* 90: RNG global interrupt */
+#define STM32_IRQ_LPUART1     (STM32_IRQ_FIRST+91)  /* 91: LPUART1 */
+#define STM32_IRQ_I2C3EV      (STM32_IRQ_FIRST+92)  /* 92: I2C3 event interrupt */
+#define STM32_IRQ_I2C3ER      (STM32_IRQ_FIRST+93)  /* 93: I2C3 error interrupt */
+#define STM32_IRQ_DMAMUXOV    (STM32_IRQ_FIRST+94)  /* 94: DMAMUX overrun interrupt */
+#define STM32_IRQ_QUADSPI     (STM32_IRQ_FIRST+95)  /* 95: QuadSPI global interrupt */
+#define STM32_IRQ_DMA1CH8     (STM32_IRQ_FIRST+96)  /* 96: DMA1 channel 8 global interrupt */
+#define STM32_IRQ_DMA2CH6     (STM32_IRQ_FIRST+97)  /* 97: DMA2 channel 6 global interrupt */
+#define STM32_IRQ_DMA2CH7     (STM32_IRQ_FIRST+98)  /* 98: DMA2 channel 7 global interrupt */
+#define STM32_IRQ_DMA2CH8     (STM32_IRQ_FIRST+99)  /* 99: DMA2 channel 8 global interrupt */
+
+#define STM32_IRQ_CORDIC      (STM32_IRQ_FIRST+100) /* 100: CORDIC trigonometric accelerator interrupt */
+#define STM32_IRQ_FMAC        (STM32_IRQ_FIRST+101) /* 101: FMAC filter math acclerator interrupt */
+
+#define STM32_IRQ_NEXTINT     (102)
+#define NR_IRQS               (STM32_IRQ_FIRST+102)
+
+/* Interrupt vectors that are shared with numbers defined above: */
```

Comment:
I think we should organize the file like other STM32 chips. So if we have a IRQ number shared between peripherals, we put the definitions one by one.

Reply:
Done in fade67cca2d31a39ad8e746da83b5c65e57e1add.